### DISCUSSION TITLE
run rt.sh on wcoss2

### DISCUSSION THREAD

**Discussion by jiandewang** [ufs-community/ufs-weather-model#2328] [Q&A] [answered]
I am testing MOM6 using rt.sh on wcoss2 but jobs failed to be launched.
The UWM is up to date and the command I am using is: ./rt.sh -e -a GFS-DEV -k -l rt.conf-S2S>output
but I am seeing:

Checking if the server is already running on cdecflow01 and port 34196
[18:44:55 17.6.2024] Request( --ping ), Failed to connect to cdecflow01:34196. After 2 attempts. Is the server running ?

Backing up check point and log files

OK starting ecFlow server...

Placing server into RESTART mode...

To view server on ecflow_ui - goto Servers/Manage Servers... and enter
Name        : <unique ecFlow server name>
Host        : cdecflow01
Port Number : 34196

ping server(cdecflow01:34196) succeeded in 00:00:00.120766  ~120 milliseconds
rt_utils.sh: Starting ECFLOW tasks...
rt_utils.sh: Total number of tasks processed -- 32

ECFLOW Tasks Remaining: 32/32 Will force aborted state for task /regtest_111091/cpld_control_p8_mixedmode_intel
Will force aborted state for task /regtest_111091/cpld_control_gfsv17_intel
Will force aborted state for task /regtest_111091/cpld_control_gfsv17_iau_intel

is it because ecflow is not working at this moment ?

my run output can be found at /lfs/h2/emc/couple/noscrub/Jiande.Wang/MOM6-update/Marshall-BBL-fix/ufs-weather-model/tests/output

**Comment by BrianCurtis-NOAA** [comment, marked as answer]
There's a bug getting fixes in the latest UFSWM PR: Please try using modulefile: /lfs/h2/emc/nems/noscrub/brian.curtis/git/ufs-community/ufs-weather-model/modulefiles/ufs_wcoss2.intel.lua

**Comment by jiandewang** [comment]
@BrianCurtis-NOAA thanks for the quick response. let me take a try

**Reply by gspetro-NOAA** [reply]
@jiandewang Are you all set? If so, I'd like to close the discussion. If not though, I can keep it open. :)

**Comment by jiandewang** [comment]
@gspetro-NOAA can you keep it open at this moment as the fixing has not been merged to develop branch yet

**Reply by gspetro-NOAA** [reply]
Sure! 👍
